test-suite/js/src/test/scala/org/scalajs/testsuite/jsinterop/StrangeNamedTests.scala

```diff
@@ -1,28 +0,0 @@
-/*                     __                                               *\
-**     ________ ___   / /  ___      __ ____  Scala.js Test Suite        **
-**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2013, LAMP/EPFL        **
-**  __\ \/ /__/ __ |/ /__/ __ |/_// /_\ \    http://scala-js.org/       **
-** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
-**                          |/____/                                     **
-\*                                                                      */
-package org.scalajs.testsuite.jsinterop
-
-import org.scalajs.jasminetest.JasmineTest
-
-object `1_TestName` extends JasmineTest { // scalastyle:ignore
```

Comment:
There are already some equivalent tests in JUnit in https://github.com/scala-js/scala-js/blob/master/test-suite/js/src/test/scala/org/scalajs/testsuite/junit/JUnitNamesTest.scala